impl/src/planning/subplanners.rs

```diff
@@ -76,6 +76,7 @@ struct DependencySet {
   aliased_deps: Vec<DependencyAlias>,
 }
 
+/** A set of dependencies that a crate has for a specific target/cfg */
```

Comment:
This change should probably not be part of this PR.

Reply:
I figured it's harmless. I can remove it if you feel strongly though. What other PR would it go in otherwise?

Reply:
I've made comment similar to this one elsewhere. Basically, I feel pretty strongly that PRs should be focused on whatever their title and description are focused on, and incidental changes should be moved to smaller PRs. This is most important when dealing with a release process where commits might get cherry picked or whatever (so, not really this project), but I still consider it good practice.

I'm generally lenient on this practice for junior engineers or folks who aren't familiar enough with git or mercurial (... such that they might shoot themselves in the foot trying to handle it...) but I've seen you deftly split PRs into smaller reviewable units, so I'd appreciate a clean separation here.

Just create a "clean up and document subplanner code" PR.

Reply:
Removed these changes and made https://github.com/google/cargo-raze/pull/264